Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -81,6 +84,14 @@ class UserScriptManager {
         }
     }
     
+    public static func isMessageHandlerTokenMissing(in body: [String: Any]) -> Bool {
+        guard let token = body["securitytoken"] as? String, token == UserScriptManager.messageHandlerToken.uuidString else {
+            log.debug("Missing required security token.")
```

Comment:
This method is called in all the regions where this log is also present. Probably best to remove it from here as it won't let us pin-point in the log itself (which includes file name/line) which helper failed security token check.

Reply:
done!